pkg/analysis_server/lib/src/services/correction/fix_internal.dart

```diff
@@ -1816,6 +1819,16 @@ class FixProcessor {
     }
   }
 
+  void _addFix_removeEmptyStatement() {
+    EmptyStatement emptyStatement = node is EmptyStatement
+        ? node
+        : node.getAncestor((node) => node is EmptyStatement);
```

Comment:
Empty statements can't have anything inside them, so I *know* this is unnecessary. Just replace the null check below with `node is EmptyStatement`.